web/ui/react-app/src/pages/targets/SeriesLabel.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
```

Comment:
I would use the Badge component without abstracted it away.

Reply:
Yeah, arguably this is a bit too much abstraction for a little thing that can just appear verbatim anywhere it's used.

Reply:
Cool, will do